### PR TITLE
docs: improve plugin docs

### PR DESCRIPTION
**Pull request by catdevnull**

### CODE REVIEW

**Review by 6543 — APPROVED** [review]

File: docs/docs/20-usage/51-plugins/10-plugins.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Plugins
 
-Plugins are Docker containers that perform pre-defined tasks and are configured as steps in your pipeline. Plugins can be used to deploy code, publish artifacts, send notification, and more.
+Plugins are pipeline steps that perform pre-defined tasks and are configured as steps in your pipeline. Plugins can be used to deploy code, publish artifacts, send notification, and more.
+
+They are automatically pulled from [plugins.drone.io](https://plugins.drone.io).
```
Comment (anbraten) [inline docs/docs/20-usage/51-plugins/10-plugins.md:5]: Domain is http only:

```suggestion
They are automatically pulled from [plugins.drone.io](http://plugins.drone.io).
```

Is it really taking the plugins from that page or is it just an index?

**Review by anbraten — CHANGES_REQUESTED** [review]

File: docs/docs/20-usage/51-plugins/10-plugins.md (hunk 1 above)
Comment (6543) [reply docs/docs/20-usage/51-plugins/10-plugins.md:5]: wait it's http ?!!

Comment (anbraten) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: Yip 😢 Maybe we should start some own marktplace with an repo anyone can share their own plugins with the community by some PR.

Comment (6543) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: yes that's a good idear! this doesnt look well maintained anymore :/

Comment (catdevnull) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: I noticed that and hoped that it was just an issue on my side... It's just an index, it takes the actual images from hub.docker.com. Example: <https://hub.docker.com/r/plugins/docker>

Comment (anbraten) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: Crazy they really claimed `plugins` org at docker-hub. Searched for days to find the image resolution step 🤦🏾

Comment (catdevnull) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: Damn, we should probably fork `plugins/git` (runs in every pipeline!) as it's unmaintained... https://github.com/drone-plugins/drone-git

Comment (catdevnull) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: > Crazy they really claimed `plugins` org at docker-hub. Searched for days to find the image resolution step 🤦🏾

Same lol

Comment (anbraten) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: see #303

**Comment by catdevnull** [comment]
Should we clarify that plugins.drone.io will be replaced?

**Comment by anbraten** [comment]
> Should we clarify that plugins.drone.io will be replaced?

We can change the docs when doing the PR for #315

**Review by anbraten — APPROVED** [review]

**Pull request merged**